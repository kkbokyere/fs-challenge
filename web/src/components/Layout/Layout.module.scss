@import "../../styles/variables";
.container {
  margin: auto;
  padding: 50px 20px;
  @include desktop {
    width: $mobile-width;
    padding: auto;
  }
}
