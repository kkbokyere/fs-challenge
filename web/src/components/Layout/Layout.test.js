import React from 'react';
import { render } from '@tests/helper'

import Layout from '@components/Layout';
describe('Layout Tests', () => {

  const setup = (props) => {
    return render(<Layout {...props}/>);
  };
  it('should render Layout', async () => {
    const { asFragment } = setup();

    expect(asFragment()).toMatchSnapshot();
  });
});
