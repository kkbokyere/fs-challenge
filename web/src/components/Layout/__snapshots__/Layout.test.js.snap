// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Tests should render Layout 1`] = `
<DocumentFragment>
  <div
    class="container"
  />
</DocumentFragment>
`;
